### DISCUSSION TITLE
configuring Issuer to run with own DID and private key EBSI

### DISCUSSION THREAD

**Discussion by anishsapkota** [walt-id/.github#85] [Help]
Hi, 
for our university project, we want to use Walt Id.
The idea is to issue a demo University Degree Credential with our uni's DID and sign it using the private key.

First question: how can we configure the issuer to issue a signed VC?
Second question: This is regarding the verifier. we noticed: Signature not verified, while presenting the existing demo UniversityDegree VC via openid4vp. 

We thank you for your help. :)

**Comment by alegomes** [comment]
This forum is no longer being monitored. Please, if you still need our help, submit your question as a [Community Stack issue](https://github.com/walt-id/waltid-identity/issues).

All the best
